packer/install-dependencies.sh

```diff
@@ -336,20 +284,50 @@ for NVIDIA_DEVICE_PLUGIN_VERSION in ${NVIDIA_DEVICE_PLUGIN_VERSIONS}; do
     echo "  - ${CONTAINER_IMAGE}" >> ${VHD_LOGS_FILEPATH}
 done
 
-TUNNELFRONT_VERSIONS="v1.9.2-v4.0.4"
+TUNNELFRONT_VERSIONS="v1.9.2-v4.0.11"
```

Comment:
We need "v1.9.2-v3.0.11 v1.9.2-v4.0.11" based on the recent change from Guoxun

https://github.com/Azure/aks-engine/pull/2546/files